code/game/objects/items/weapons/storage/book.dm

```diff
@@ -42,7 +42,8 @@
 			if(affecting.status == ORGAN_ORGANIC) //No Bible can heal a robotic arm!
 				if(affecting.heal_damage(heal_amt, heal_amt, 0))
 					H.update_damage_overlays(0)
-	return
+		return 1
```

Comment:
This is going to return 1 for any human, since it's outside of the loop. I'd suggest setting `. = 0` at the beginning of the proc (`.` being the default return value), and then `. = 1` in the for loop, so that you return `1` only if any body part was actually healed.